ctapipe/reco/HillasReconstructor.py

```diff
@@ -99,10 +98,14 @@ class that reconstructs the direction of an atmospheric shower
 
     """
 
-    def __init__(self, config=None, parent=None, **kwargs):
+    def __init__(self, event=None, config=None, parent=None, **kwargs):
         super().__init__(config=config, parent=parent, **kwargs)
         self.hillas_planes = {}
-        self.divergent_mode = False
+        self.mode = "CameraFrame"
```

Comment:
If this PR is about enabling both, shouldn't this be a config option instead of hard coding `CameraFrame` here?

Reply:
technically we could get rid of CameraFrame, but for test purposes we should enabel both modes.
this should then become an argument to init if I understand your point